### PR TITLE
Add cubeFS CSI driver to Production Driver.

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #493 

Replace [ChubaoFS] CSI driver to CubeFS CSI driver in Production Drivers.

### CODE REVIEW

**Comment by Kartik494** [comment]
cc @xing-yang . PTAL

File: book/src/drivers.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [CephFS](https://github.com/ceph/ceph-csi) | `cephfs.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI) Driver for CephFS | Persistent | Read/Write Multiple Pods | Yes | Expansion, Snapshot, Cloning
 [Ceph RBD](https://github.com/ceph/ceph-csi) | `rbd.csi.ceph.com` | v0.3, >=v1.0.0 | A Container Storage Interface (CSI)  Driver for Ceph RBD | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Topology, Cloning,In-tree plugin migration
 [Cisco HyperFlex CSI](https://github.com/ciscohyperflex/csi-plugin) | `HX-CSI` | v1.2 | A Container Storage Interface (CSI) Driver for Cisco HyperFlex | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Expansion, Cloning
-[ChubaoFS](https://github.com/chubaofs/chubaofs-csi) | `csi.chubaofs.com` | v1.0.0 | A Container Storage Interface (CSI) Driver for ChubaoFS Storage  | Persistent | Read/Write Multiple Pods | Yes | 
+[CubeFS](https://github.com/cubefs/cubefs-csi) | `csi.cubefs.com` | v2.4.1 | A Container Storage Interface (CSI) Driver for CubeFS Storage  | Persistent | Read/Write Multiple Pods | Yes |
```
Comment (xing-yang) [inline book/src/drivers.md:32]: The 3rd column is CSI spec version, not driver version.

Comment (Kartik494) [reply book/src/drivers.md:32]: @xing-yang sorry my bad i will update this, thanks for  the information.

Comment (Kartik494) [reply book/src/drivers.md]: @xing-yang  i have addressed the comments as update the CSI spec version mentioned in [cube fs docs](https://github.com/cubefs/cubefs-csi#prerequisite) .

**Review by saad-ali — APPROVED** [review]
/lgtm
/approve

**Pull request merged**